### ISSUE TITLE
Text alternative does not present same info 

### ISSUE THREAD

**Issue by shobhit-v** [shobhit-v/demo#90]
## Issue Description
**Impact:** Serious
**Issue Summary:** Text alternative does not present same info 
**Issue Description:** The text alternative does not present the same information as the image.

**WCAG Standards:** wcag2a, wcag2aa, wcag21a, wcag21aa
**Test Case Name:** Test web assessment
**Test Run Name:** Test web assessment_07-27-21-12:23
**Checkpoint Group:** 1.1 Text Alternatives
**Issue Type:** Accessibility
**Unit Type:** Page
**Test Unit:** Test page
**Recommended to fix:**
 `RULE : 
The alternative text for informative images MUST be accurately conveying the purpose of the image, and the author's intent in a way that is useful to those who cannot see the image.

HOW TO FIX: 
Fix this issue by ensuring that the alternative text for the informative image describes its intent, purpose, or meaning.

<img src="sunset.jpg" alt="The sun setting over the Pacific Ocean, with a silhouette of a flying seagull in the foreground">

REFERENCE: 
Deque University: https://dequeuniversity.com/class/images2/alt-text/informative
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/images/

BACKGROUND: 
People who are blind cannot see images on a page. In order to give people who cannot see an image access to the information conveyed by the image, it must have a text alternative. The text alternative must describe the information or function represented by the image. Screen readers can then use the alternative text to convey that information to the screen reader user.

For informative images - images that convey information - the alternative text must communicate the intent, purpose, or meaning of the image in a way that serves as a true alternative for the image.`
**Digital Asset Type:** Desktop Web
**Release:** g
**Environment:** csv
**Platform:** cvsa
**Assistive technology:** s
**Checkpoint:** Alternative Text (Informative Images) (1.1.1.b)
**Method:** Manual
**Tested On URL:** www.google.com
